client/my-sites/sidebar/style.scss

```diff
@@ -422,6 +422,11 @@ $font-size: rem(14px);
 
 		.upsell-nudge.banner.card.is-compact .banner__info .banner__title {
 			color: #000;
+			text-wrap: wrap;
+		}
+
+		.upsell-nudge.banner.card.is-compact .dismissible-card__close-button .gridicon {
+			fill: #000;
```

Comment:
Is there a css var we should be using here instead of `#000`? 🤔  I noted it was already used a few lines above, and there's quite a few references to `#000` around the codebase, so maybe not?

Reply:
Great question :) 
We do have a centralized palette package which we should have referred to whenever possible: https://github.com/Automattic/color-studio. In this case, what you need is `--studio-black`.

Reply:
Perfect, thanks! Updated: a3b6fdb946d7e3eaabf7f79226f816ca99993e38